tests/integration/galley/webhook_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package galley
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+)
+
+var (
+	vwcName    = "istio-galley"
+	deployName = "istio-galley"
+	istioNs    = "istio-system"
```

Comment:
You can obtain this from settings of an Istio deployment.

Reply:
+1, this is not guaranteed

Reply:
Would you mind pointing me in the right direction on how I should do this?

Reply:
I've figured out how to get the namespace (via `Settings().istioNamespace` on the `istio.Instance` object) but I'm not sure how I should get the name of the ValidatingWebhookConfig object or the galley deployment.

Reply:
I think hardcoding them for the time being is ok. You can get this from configmap files, but that is not necessarily a more useful thing to do.